src/bridge.c

```diff
@@ -68,3 +68,26 @@ size_t sizeof_tspoint() {
 size_t sizeof_node() {
   return sizeof(Node);
 }
+
+size_t sizeof_tstreecursor() {
+  return sizeof(TSTreeCursor);
+}
+
+
+void ts_tree_cursor_new_p(TSNode *node, TSTreeCursor *outCursor) {
+  assert(node != NULL);
```

Comment:
If we hit this assert, the calling Haskell program will die with what I believe is an unrecoverable exception. That might be okay for this case; what do you think? If necessary, we can wrap this in a function that throws some recoverable exception based on a return code.

Reply:
Thus far we’ve done this pattern for everything bridged, and I’d like to avoid changing it in this PR. Let’s file an issue about that.

Reply:
Filed as #89.